packages/main/src/ToolbarButton.ts

```diff
@@ -0,0 +1,124 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+
+import ButtonDesign from "./types/ButtonDesign.js";
+
+import ToolbarItem from "./ToolbarItem.js";
+import ToolbarButtonTemplate from "./generated/templates/ToolbarButtonTemplate.lit.js";
+import ToolbarPopoverButtonTemplate from "./generated/templates/ToolbarPopoverButtonTemplate.lit.js";
+
+@customElement({
+	tag: "ui5-toolbar-button",
+})
+
+/**
```

Comment:
Check how we describe public events and add description. Also we need to move that decorator after class JSDoc